node/cli_tools.go

```diff
@@ -95,14 +95,22 @@ func FlagsToArgs(flags Flags, basedir string, sepBase bool) ([]string, Metadata)
 		"--network-initial-timeout=" + flags.NetworkInitialTimeout,
 		"--network-minimum-timeout=" + flags.NetworkMinimumTimeout,
 		"--network-maximum-timeout=" + flags.NetworkMaximumTimeout,
+		"--network-health-max-time-since-no-requests=" + flags.NetworkHealthMaxTimeSinceNoReqsKey,
+		fmt.Sprintf("--network-health-max-send-fail-rate=%f", flags.NetworkHealthMaxSendFailRateKey),
```

Comment:
Would it make sense to standardize concatenating the strings vs. using `fmt.Sprintf(...)` as part of this PR? I personally think using `fmt.Sprintf(...)` is a little bit cleaner.